cmd/podman/containers/run.go

```diff
@@ -86,6 +90,7 @@ func runFlags(cmd *cobra.Command) {
 
 	if registry.IsRemote() {
 		_ = flags.MarkHidden("preserve-fds")
+		_ = flags.MarkHidden("preserve-fd")
```

Comment:
And why does this use the string instead of the variable? (line 92 also). I don't actually see a strong need for a variable in lines 73-75, but since there is one, might as well use it here too?